wire-ios-data-model/Source/MLS/Actions/UpdateConversationProtocolAction.swift

```diff
@@ -0,0 +1,55 @@
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+public struct UpdateConversationProtocolAction: EntityAction {
+
+    // MARK: - Properties
+
+    public var domain: String
+    public var conversationID: UUID
+    public var messageProtocol: MessageProtocol
+    public var resultHandler: ResultHandler?
+
+    // MARK: - Life cycle
+
+    public init(
+        domain: String,
+        conversationID: UUID,
```

Comment:
Should these two arguments be combined into one `qualifiedID: QualifiedID` argument?

Reply:
I'd say yes